src/store/reducers/nodes/nodes.ts

```diff
@@ -99,9 +99,9 @@ const nodes: Reducer<NodesState, NodesAction> = (state = initialState, action) =
 };
 const concurrentId = 'getNodes';
 
-export function getNodes({type = 'any', ...params}: NodesApiRequestParams) {
+export function getNodes({type = 'any', storage = false, ...params}: NodesApiRequestParams) {
```

Comment:
Explicitly set `storage=false` flag for nodes request (it's implicitly `false` by default)